proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -204,3 +209,23 @@ message MsgUpdateParams {
 //
 // Since: cosmos-sdk 0.47
 message MsgUpdateParamsResponse {};
+
+// MsgRotateConsPubKey is the Msg/RotateConsPubKey request type.
+//
+// Since: cosmos-sdk 0.51
+message MsgRotateConsPubKey {
+  option (cosmos.msg.v1.signer) = "validator_address";
+  option (amino.name)           = "cosmos-sdk/MsgRotateConsPubKey";
+
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.equal)           = false;
+
+  string              validator_address = 2 [(cosmos_proto.scalar) = "cosmos.ValidatorAddressString"];
+  google.protobuf.Any new_pubkey        = 3 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
```

Comment:
ditto

Reply:
Yes this is also needed, this is the new pubkey that the validator want to change to.